code/game/objects/items.dm

```diff
@@ -443,10 +443,11 @@ var/global/image/fire_overlay = image("icon" = 'icons/goonstation/effects/fire.d
 
 	playsound(loc, src.hitsound, 30, 1, -1)
 
+	user.do_attack_animation(M)

```

Comment:
reason this was moved is there's logic within `do_attack_animation` that returns if the atom being attacked is doing the attack on itself. Well, the moving part of the attack; the item/unarmed animation will play no matter what, as that's still important info to communicate.